### ISSUE TITLE
Grouped Convolution Block

### ISSUE THREAD

**Issue by carlosrivera22** [titu1994/Keras-ResNeXt#14]
I'm not sure about the implementation of the grouped convolution block. The attached image shows a different approach if I'm not mistaken. The repository has only one convolution per tower while the image shown has 3 convolutions followed by BatchNormalization and Relu, except for the last one which is only followed by BatchNormalization. 

Image found in: https://towardsdatascience.com/illustrated-10-cnn-architectures-95d78ace614d 
![1_HelCJiQZEuwuKakRwDdGPw](https://user-images.githubusercontent.com/14832117/65072080-2b698680-d95e-11e9-9a9a-6deed4f68af9.png)
